### ISSUE TITLE
What should I do if I want to train VIPeR

### ISSUE THREAD

**Issue by dayby** [VisualComputingInstitute/triplet-reid#38]
Hello, I want to train VIPeR, and  set the batch-k to 2,but when I test, the top-1 is 25.33%. I want to know did you trained the VIPeR?Thank you

**Comment by Pandoro** [comment]
We never tried it on the viper dataset. As far as I recall, since viper only has 2 images per person, the whole idea behind our PK batches is a little irrelevant. Additionally since it is a tiny dataset we did not consider it. I assume it is trivial for a resnet-50 to completely fit the training set without any real generalization capabilities. If you really really need results on viper, I would train on Market-1501 and then fine-tune on viper. But you should most definitely adjust the training schedule since the dataset is a lot smaller and 25k iterations for 200 images would be an overkill.

In my honest opinion, the results on VIPER are not very relevant anymore. The only way I would use the dataset would be to evaluate the generalization of an approach.

**Comment by lucasb-eyer** [comment]
Yup, and in addition, please do not simply ignore this line of the README:

:boom: :fire: :exclamation: **If you train on a very different dataset, don't forget to tune the learning-rate** :exclamation: :fire: :boom:

Actually, I'll amend it to add that also the schedule is just as important.

We have never tried VIPeR ourselves and as @Pandoro says are not too interested in it, but if anybody else had success with it, they are welcome to share their experiences.

**Comment by ss199302** [comment]
@dayby @lucasb-eyer  Hello, can you provide a weight model for pre-training? Thank you!

**Comment by Pandoro** [comment]
@ss199302 as I told you in the other issue, everything is in the ReadMe.

We use the original tf slim pretrained model which you can download from their github account, the link is in our ReadMe. Also our already trained model for Market-1501 is linked in the ReadMe.

Please do not post this under unrelated issues. In the future I will see myself forced to delete this comments to keep issues on topic.
